app/components/rdf-form-fields/validation-concept-scheme-selector.js

```diff
@@ -147,7 +149,38 @@ export default class ValidationConceptSchemeSelectorComponent extends InputField
       updateSimpleFormValue(this.storeOptions, option.subject);
```

Comment:
I would remove this and update both the type and grouping at the same time (addAll + removeStatements call). That will trigger the observers "only" twice (until I add a new method to the forking store to do both inserts and deletes in a single method call).

Reply:
Adding the rdf type and the form grouping in one addAll([]) call